### PR TITLE
Speech bubbles on subscriptions

### PR DESCRIPTION
**Pull request by alexyork**

### CODE REVIEW

File: src/components/molecules/Subscription/Subscription.jsx [hunk 1]
```diff
@@ -63,6 +63,9 @@ export default class Subscription extends React.Component {
                 </section>
                 {(this.state.isExpanded || this.props.isShowingFeatures) ?
                     <section className="subscription__features">
+                        {this.props.speechBubbleText
+                            ? <div className="subscription__speech-bubble">{this.props.speechBubbleText}</div>
+                            : <div className="subscription__speech-bubble subscription__speech-bubble--empty"></div>}
```
Comment (cathrholm) [inline src/components/molecules/Subscription/Subscription.jsx]: I can see why this is done, but in general I don't think we should render html if it's not supposed to be visual. Screen readers read elements that has visibility hidden.

Comment (alexyork) [reply src/components/molecules/Subscription/Subscription.jsx]: I agree, it's a small hack. How can we solve this in another way? We want the feature to be in the same place regardless of what is above it. I tried absolute positioning but it was tricky. Do you wanna fix? :-)

Comment (cathrholm) [reply src/components/molecules/Subscription/Subscription.jsx]: Maybe we can fix this when we refactor subscription to use the Box component? I think we'll have to make adjustments anyway at that point :-)

Comment (alexyork) [reply src/components/molecules/Subscription/Subscription.jsx]: Ja!

**Pull request merged**